MembraneRTC/Sources/MembraneRTC/MembraneRTC.swift

```diff
@@ -338,20 +364,93 @@ public class MembraneRTC: MulticastDelegate<MembraneRTCDelegate>, ObservableObje
             self.config.iceServers = [Self.defaultIceServer()]
         }
 
-        guard let peerConnection = ConnectionManager.createPeerConnection(config, constraints: Self.mediaConstraints) else {
+        guard let peerConnection = connectionManager.createPeerConnection(config, constraints: Self.mediaConstraints) else {
             fatalError("Failed to initialize new PeerConnection")
         }
         connection = peerConnection
 
         peerConnection.delegate = self
         
         localTracks.forEach { track in
-            peerConnection.add(track.rtcTrack(), streamIds: [localStreamId])
+            if(track.rtcTrack().kind == "video" && (track as? LocalVideoTrack)?.simulcastConfig.enabled == true) {
+                let simulcastConfig = (track as? LocalVideoTrack)?.simulcastConfig
+                
+                let sendEncodings = [
+                    RTCRtpEncodingParameters.create(rid: "l", active: false, scaleResolutionDownBy:4.0),
+                    RTCRtpEncodingParameters.create(rid: "m", active: false, scaleResolutionDownBy:2.0),
+                    RTCRtpEncodingParameters.create(rid: "h", active: false, scaleResolutionDownBy:1.0),
+                ]
```

Comment:
Would it be possible to move this to e.g. module with constants so that searching for this and chaning it in the future will be easier. Something like we have in `js` repository (`consts.ts`).